index.js

```diff
@@ -49,60 +57,148 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-app.get('/', function (req, res) {
-    res.render('index', {
-        chainId: chainId,
-        denom: denom
-    });
-});
-
-app.post('/airdrop', (req, res) => {
-    let ip = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
-
+//fuction check if the address has a validator with voting power higher than defined retriction rule 
+const gValidator = async(desmosAddress) => {
+    let url1 = lcdAddress+"/validatorsets/latest";
```

Comment:
Just noticed this. This won't work as it will return addresses with `desmosvalcons` prefix. What we need to check is whether the account address (`desmos` prefix) is delegating to any validator operator address (`desmosvaloper` prefix). The list can be retrieved with `/staking/validators` endpoint.